Makefile

```diff
@@ -90,6 +90,15 @@ docker-images: Dockerfile learn-ocaml.opam
 	@docker build -t learn-ocaml --target program docker
 	@echo "Use with 'docker run --rm -v \$$PWD/sync:/sync -v \$$PWD:/repository -p PORT:8080 learn-ocaml -- ARGS'"
 
+# Generates documentation for testing (exclusively Test_lib and Learnocaml_report modules)
+doc: build
```

Comment:
This target does not seem to be declared in the .PHONY.

Reply:
Just learn what .PHONY is. I push something (!). Is that what you mean @yurug ?